styles/components/_modeMenu.scss

```diff
@@ -0,0 +1,40 @@
+.menu {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  max-width: 1200px;
+  height: 180px;
+  border-radius: 16px;
+  background-color: #ffffff;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+    4px 4px 40px rgba(255, 0, 0, 0.9);
+  &_button {
+    position: relative;
+    border: none;
+    padding-bottom: 20px;
+    background-color: #ffffff;
+    color: rgba(0, 0, 0, 0.4);
+    font-family: $font-primary;
+    text-align: center;
+    font-size: 36px;
+    color: rgba(0, 0, 0, 0.4);
+    font-weight: 600;
+    outline: none;
+    cursor: pointer;
+  }
+  &_button--select {
```

Comment:
Maybe the structure could look like that: 
.menu {
  &__button {
    &--select{
      &::after{
      }
    }